optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/employee/Employee.java

```diff
@@ -45,6 +46,7 @@
 
     @NotNull
     @Size(min = 1, max = 120)
+    @Pattern(regexp = "^\\S.*\\S$", message = "Name should not contain any leading or trailing whitespaces")
```

Comment:
This pattern forces names to be at least two chars (both "\S" need to match a char). I don't see an immediate way to get around this just using regex, so I recommend adding a method annotated with `@AssertTrue` which returns `name.equals(name.trim())`.

Reply:
The @AssertTrue isn't showing the feedback message, instead it's displaying an error from the REST api.